pkg/apis/storage/types.go

```diff
@@ -72,6 +72,12 @@ type StorageClass struct {
 	// the VolumeScheduling feature.
 	// +optional
 	VolumeBindingMode *VolumeBindingMode
+
+	// Snapshotter is the driver expected to handle this StorageClass.
+	// This is an optionally-prefixed name, like a label key.
+	// For example: "kubernetes.io/gce-pd" or "kubernetes.io/aws-ebs".
+	// This value may not be empty.
+	Snapshotter string
```

Comment:
The driver for Provisioner and Snapshotter should be the same.  Can we just use Provisioner?

Reply:
I'll merge this now.  Please take a look and see if we can just use Provisioner.

Reply:
Sorry for the delay, I replay this last friday, but the message is always `pending`, replay this again.
>Provisioner and Snapshotter are same and are just the plugin's name when we use in-tree plugin. out of tree Provisioner and Snapshotter can define their own name and maybe not same. We can use the Provisioner field as the snapshotter name,but I am afraid that it may confuse the user.